roles/devscripts/tasks/sub_tasks/14_user.yml

```diff
@@ -50,8 +50,10 @@
   when:
     - cifmw_devscripts_config['network_type'] == 'OVNKubernetes'
     - "cifmw_devscripts_config.openshift_version is ansible.builtin.version('4.14.0', '>=')"
-  ansible.builtin.copy:
-    src: "files/forwarding.yaml"
+  vars:
+    host_routing: "{{ cifmw_devscripts_host_routing }}"
+  ansible.builtin.template:
+    src: "templates/forwarding.yaml.j2"
```

Comment:
would be good to use different name than forwarding.yaml as scope is now increased to also set routeviahost conditionally. Something like patch_networkoperator.yaml ?

Reply:
Routingviahost need to be set even for versions <4.14.0

Reply:
Thank you for the clarification.

Yes, the file can be renamed. Let me rename it to `net_op_defaults.yaml`. `patch` may be misleading as we are providing the custom configuration during installation.